src/azure-cli/azure/cli/command_modules/serviceconnector/custom.py

```diff
@@ -848,6 +888,9 @@ def local_connection_update_kafka(cmd, client,  # pylint: disable=too-many-local
                                           resource_group_name=resource_group_name,
                                           location=location,
                                           connector_name=connection_name))
+        if server_linker.get('configurationInfo') and server_linker.get('configurationInfo').get('customizedKeys'):
+            customized_keys = customized_keys or server_linker.get('configurationInfo').get('customizedKeys')
+
```

Comment:
could be simplified as `customized_keys = customized_keys or server_linker.get('configurationInfo', dict()).get('customizedKeys', dict())`